pep_sphinx_extensions/pep_theme/static/style.css

```
@@ -248,20 +245,20 @@ dl.footnote {
     line-height: 1.875;
     width: 100%;
     border-top: 1px solid var(--colour-background-accent);
```

```suggestion
    border-top: 1px solid var(--colour-rule-strong);
```

To match other comment.